src/style.css

```diff
@@ -0,0 +1,27 @@
+* {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin-top: 5px;
+}
+
+h2 {
```

Comment:
`h2` must have vertical spacing `1em`

Reply:
when all vertical spacing have zero values, except h2, a half of tests were failed, so I left this unchanged